resources/scss/componets/08_about.scss

```diff
@@ -0,0 +1,229 @@
+        }
+        p{
+            font-size: 15px;
+            font-weight: 400;
+            font-family: 'Khula', sans-serif;
+            color: $grayish;
+            line-height: 2;
+            margin: 0;
+        }
+    }
+}
+
+/*about welcome section end  */
+
+
+/*about agency start*/
+.intro-section {
+    box-shadow: 0px 0px 200px rgba(0, 0, 0, 0.05) inset;
+    background-size: auto;
+    background-repeat: repeat;
+
+    .intro-img {
+        position: relative;
+        margin-top: -170px;
+        padding-right: 50px;
+
+        img {
+            border: 10px solid $white;
+            box-shadow: 0px 10px 40px -4px rgba(0, 0, 0, 0.1);
+        }
+
+        .vid-btn{
+            position: absolute;
+            right: 20px;
+            bottom: 50px;
+        }
+    }
+
+    .intro-content{
+        .intro-title{
+            margin-bottom: 15px;
+            font-weight: 600;
+            font-size: 40px;
+            line-height: 1.2;
+            @media screen and (max-width: 767px) {
+                font-size: 25px;
+            }
+        }
+        .desc{
+            font-size: 15px;
+            font-weight: 400;
+            font-family: 'Khula', sans-serif;
+            color: $grayish;
+            line-height: 2;
+            margin: 0;
+        }
+
+        .skills{
+            margin-top: 50px;
+            .each-skill{
+                h5{
+                    font-size: 14px;
+                    margin-bottom: 10px;
+                    span{
+                        display: inline-block;
+                        text-decoration: none;
+                        color: inherit;
+                        float: right;
+                        font-size: 12px;
+                    }
+                }
+
+
+            }
+        }
+    }
+}
+/*about agency end  */
+
+
+/*about service start*/
+.service-section {
+    .each-service {
+        margin-bottom: 50px;
+
+        .icon {
+            font-size: 40px;
+            background: linear-gradient(45deg, $theme 0%, $yellow 100%);
+            -webkit-background-clip: text;
+            -webkit-text-fill-color: transparent;
+            float: left;
+        }
+
+        .service-title {
+            margin-left: 70px;
+            padding-bottom: 20px;
+            margin-bottom: 20px;
+            position: relative;
+            font-weight: 500;
+
+            &:after {
+                content: '';
+                width: 40px;
+                height: 1px;
+                background: $gray-2;
+                position: absolute;
+                left: 0;
+                bottom: 0;
+            }
+
+        }
+
+        .service-desc {
+            margin-left: 70px;
+            font-size: 14px;
+            font-family: 'Khula', sans-serif;
+            color: $grayish;
+            line-height: 2;
+            margin-bottom: 0;
+            word-break: break-word;
+            width: 65%;
+        }
+    }
+}
+/*about service end  */
+
```

Comment:
Move this to **resources/scss/componets/10_services.scss**

Reply:
Fixed